### PR TITLE
Use GitHub Actions instead of Travis CI and Circle CI

### PR DESCRIPTION
**Pull request by stefanvermaas**
It's easier for everyone if we just use Github Actions as our CI. It will allow everyone to inspect the things that are going potentially wrong etc.

1. Removes Circle CI as the main CI.
2. Removes Travis CI as it wasn't in use.
3. Adds GitHub Actions to run all tests before submitting the PR.

### CODE REVIEW

**Comment by stefanvermaas** [comment]
All tests will succeed after https://github.com/knowndecimal/fulfil/pull/20 is merged. We will wait until that PR is approved and merged.

File: README.md
Comment (cdmwebs) [inline README.md:3]: GH has a badge!

`example workflow`

**Review by cdmwebs — APPROVED** [review]

File: README.md
Comment (stefanvermaas) [reply README.md:3]: I'll add that one!

**Pull request merged**
